src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1090,7 +1090,35 @@ def extract_assay_expression(args):
     # assay names listing
     descriptor = DXDataset(dataset_handler.dataset, project=dataset_handler.datasetRecordProject).get_descriptor()
     dataset_handler.populate_descriptor(descriptor)
-    print(dataset_handler.list_assay_names("molecular_expression"))
+    
+    assay_names_list = dataset_handler.get_assay_names("molecular_expression")
+
+    if parser_dict.get("list_assays"):
+        print(assay_names_list)
```

Comment:
There may be more than one assays, which would be nicer to print on separate lines. 

```suggestion
        print(*assay_names_list, sep="\n")
```